src/extension.ts

```diff
@@ -107,6 +109,13 @@ export async function activate(context: ExtensionContext) {
   );
 }
 
+export function deactivate(): Thenable<void> | undefined {
+  if (!currentClient) {
+    return undefined;
+  }
+  return currentClient.stop();
```

Comment:
return currentClient?.stop()
```suggestion
  return currentClient?.stop();
```

Reply:
Fixed, I haven't noticed I could tidy it up further. Thanks!